vispy/visuals/gridlines.py

```diff
@@ -10,6 +10,9 @@
 
 
 _GRID_COLOR = """
+uniform vec4 u_gridlines_bounds;
+uniform float u_boarder_width;
```

Comment:
```suggestion
uniform float u_border_width;
```

And all uses of the word.